Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.py

```diff
@@ -0,0 +1,862 @@
+                                        file_entry=file_entry,
+                                        file_name=file_name,
+                                        invisible_for_caller=invisible_for_caller,
+                                        file_description=args.get('file_description', None))
+
+    if not response.get("downloadUrl", None):
+        raise Exception(f"Failed uploading file: {response}")
+
+    headers = ['id', 'file name', 'download url', 'size', 'description',
+               'invisible for caller', 'entry date', 'operator']
+    half_camelized_headers = [half_camelize(header, ' ') for header in headers]
+    readable_attachment: List[Dict[str, Any]] = [{}]
+    for header, half_camelized_header in zip(headers, half_camelized_headers):
+        readable_attachment[0][header] = response.get(half_camelized_header, None)
+
+    readable_output = tableToMarkdown(f"{INTEGRATION_NAME} attachment of incident {args.get('number', None)}",
+                                      readable_attachment,
+                                      headers=headers,
+                                      removeNull=True)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_NAME}.attachment',
+        outputs_key_field='id',
+        outputs=response
+    )
+
+
+''' FETCH & MIRRORING COMMANDS'''
+
+
+def fetch_incidents(client: Client,
+                    last_run: Dict[str, Any],
+                    demisto_params: Dict[str, Any]) -> Tuple[Dict[str, Any], List[Dict[str, Any]]]:
+    """Fetches incidents from TOPdesk."""
+
+    first_fetch_datetime = dateparser.parse(demisto_params.get('first_fetch', '3 days'))
+    last_fetch = last_run.get('last_fetch', None)
+
+    if not last_fetch:
+        if first_fetch_datetime:
+            last_fetch_datetime = first_fetch_datetime
+        else:
+            raise Exception("Could not find last fetch time.")
+    else:
+        last_fetch_datetime = dateparser.parse(last_fetch)
+
+    latest_created_time = last_fetch_datetime
+    incidents: List[Dict[str, Any]] = []
+
+    topdesk_incidents = get_incidents_with_pagination(client=client,
+                                                      max_fetch=int(demisto_params.get('max_fetch', 10)),
+                                                      query=demisto_params.get('query', None),
+                                                      modification_date_start=demisto_params.get(
+                                                          'modification_date_start', None),
+                                                      modification_date_end=demisto_params.get(
+                                                          'modification_date_end', None))
+
+    for topdesk_incident in topdesk_incidents:
+        if topdesk_incident.get('creationDate', None):
+            creation_datetime = dateparser.parse(topdesk_incident['creationDate'])
```

Comment:
I think we checked but just to make sure - does this come in UTC? or is this connected to a timezone?

Reply:
I think it has the option info of the timezone as well but the client uses UTC. This is the format used:
![Screen Shot 2021-03-14 at 11 01 04](https://user-images.githubusercontent.com/72339665/111062907-93961b80-84b4-11eb-9782-d35f3b4d97c9.png)

I think dateparser will handle both